sos/report/plugins/ceph_rgw.py

```diff
@@ -21,6 +22,29 @@ class CephRGW(Plugin, RedHatPlugin, UbuntuPlugin):
 
     def setup(self):
         all_logs = self.get_option("all_logs")
+        cmds = [
+            'bucket limit check',
+            'bucket list',
+            'bucket stats',
+            'datalog list',
+            'datalog status',
+            'gc list',
+            'lc list',
+            'log list',
+            'metadata sync status',
+            'period list',
+            'realm list',
+            'reshard list',
+            'sync error list',
+            'sync status',
+            'zone get --rgw-zone=default',
+            'zone list',
+            'zone placement list',
+            'zonegroup get --rgw-zonegroup=default',
```

Comment:
I see here we're only looking at `rgw-zone=default` and `rgw-zonegroup=default`, is it worth looking at others if they exist?

Reply:
I think so. Usually default is only one. But there can be others too, so expanded.